Clinical-Genomics/cgweb
# made pools searchable

How to test:
0. install branch on stage clinical-api and cgweb
1. go to OP stage
2. login
3. go to Items->pools
4. type 1 in searchbox at bottom of page and press enter
Exptected outcome:
Only those pools with a 1 in either order name or pool name should be visible.
Test also search and clear buttons
